registry/multi_master.go

```diff
@@ -0,0 +1,46 @@
+package registry
+
+import (
+	"fmt"
+	"os"
+)
+
+/*
+MultiMasterRegistry is an implementation of Registry in which multiple instances
+of an application are able to accept requests at the same time. Internally, this
+registry uses GroupMasterRegistry by creating a unique group ID for each
+instance of an application.
+*/
+type MultiMasterRegistry struct {
```

Comment:
I would call it simply a `LBRegistry` as it suits better for load balancing use-case